services/app-api/src/domain-models/nullstoUndefined.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ Recursively replaces all nulls with undefineds
+ GQL return <Maybe> types are T | null instead of T | undefined which match our zod .optional() domain types
```

Comment:
@macrael I looked into changing this via codegen settings first in the [`maybeValue`](https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-operations#maybevalue) but I was still seeing type issues. Feels like this could be something to look at more holistically but doing this again (what we did in `toDomain` with protos to quickly get around it .